<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDevices</class>
 <widget class="QDialog" name="ConfigureDevices">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hemera Static Targets</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/images.qrc">
    <normaloff>:/icons/hemera.png</normaloff>:/icons/hemera.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="targetsListWidget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="scanButton">
         <property name="text">
          <string>Scan</string>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>&amp;Add Manually</string>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editButton">
         <property name="text">
          <string>&amp;Edit</string>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/icons/document-edit.png</normaloff>:/icons/document-edit.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>&amp;Remove</string>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="actionButtonsBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
